### DISCUSSION TITLE
Textadept 11.3 Beta 3 issues on windows 11

### DISCUSSION THREAD

**Discussion by aryajur** [orbitalquark/textadept#167] [Q&A]
I downloaded and Textadept 11.3 Beta 3. I also tried 11.2. When I do view:set_theme. The theme seems to get applied to the command entry box and not to the view. 
Running that from init (running reset() on command entry) produces unexpected results on other tabs.

**Comment by orbitalquark** [comment]
Thanks for the report. I will look into this when I have some time.

**Comment by orbitalquark** [comment]
Fixed via https://github.com/orbitalquark/textadept/commit/c40d56cde2473a52e00f263c484dc9b1e80bd380
